AndroidHomeApplication/app/src/main/java/com/example/androidhomeapplication/fragments/movieDetails/FragmentMovieDetails.kt

```diff
@@ -48,6 +45,39 @@ class FragmentMovieDetails : Fragment(R.layout.fragment_movie_details) {
         }
     }
 
+    private fun setupObservers() {
+
+        viewModel.movie.observe(
+            viewLifecycleOwner,
+            {result: DataResult<Movie> ->
+                when (result) {
+                    is DataResult.Default -> {
+                    }
+                    is DataResult.Loading -> {
+                    }
+                    is DataResult.Success<Movie> -> {
+                        setMovieFields(result.value)
+                    }
+                    is DataResult.EmptyResult -> {
+                        Toast.makeText(
```

Comment:
create extension for fragment:

Toast.makeText(
                             requireContext(),
                          ......  ,
                             Toast.LENGTH_SHORT
                         ).show()